build-system/global-configs/canary-config.json

```diff
@@ -45,7 +45,6 @@
   "amp-date-picker": 1,
   "linker-meta-opt-in": 1,
   "amp-carousel-scroll-snap": 1,
-  "amp-carousel-new-arrows": 0.01,
```

Comment:
This removal from `canary-config.json` needs to be a standalone PR, preferably merged before the removal from `experiments.js`.

Reply:
Done in a separate PR.